public/locale/en/config.json

```diff
@@ -115,6 +115,8 @@
     "SR5.ArmorValue": "Armor Value",
     "SR5.AttackDodged": "Attack Dodged",
     "SR5.AttackHits": "Attack Hits",
+    "SR5.AttackBlockedByVehicleArmor": "Attack Hits But Is Blocked By Vehicle Armor",
```

Comment:
These strings could probably use some improvement

Reply:
Whenever possible group new labels:
```JSON
"SR5.Attack": {
  "BlockedByVehicleArmor": "..."
}
```

or put this message / warning into the SR5.Warnings group. Maybe we have to add a SR5.Info or Message group as well.

Reply:
Good idea - I found a `SR5.TestResults` grouping, so I moved the new strings into it and refactored all the other test result strings into it as well - I hope that is good placement.  I considered further nesting by grouping by test type like `SR5.TestResults.PhysicalDefense.BlockedByVehicleArmor` - I initially decided against it, but I could be swayed in either direction.

Reply:
`SR5.TestResults` is just the place. Didn't even remember it. I don't think further nesting is necessary, as I don't expect doubled labels.